source/codegen/templates/proto_helpers.mako

```diff
@@ -26,8 +26,9 @@ enum ${common_helpers.get_attribute_enum_name(group_name, sub_group, config)} {
 %   for attribute in attributes:
 <%
    attribute_name = attributes[attribute]["name"]
+   signed_attribute_value = attribute if attribute <= 0x7FFFFFFF else attribute - 2**32
```

Comment:
I think all the changes in this file go away if we go with representing numbers either with negative sign in metadata or with `'generate-mappings': True`

Reply:
Yep, reverted this file.